server/src/main/java/org/spine3/server/aggregate/Aggregate.java

```diff
@@ -182,16 +183,17 @@ private void createBuilder() {
     protected B getBuilder() {
         if (this.builder == null) {
             throw new IllegalStateException(
-                    "Builder is not available. Make sure to call getBuilder() only from an event applier method.");
+                    "Builder is not available. Make sure to call getBuilder() " +
+                    "only from an event applier method.");
         }
         return builder;
     }
 
     /** Updates the aggregate state and closes the update phase of the aggregate. */
     private void updateState() {
         @SuppressWarnings("unchecked") // It is safe to assume that correct builder type is passed to aggregate,
-         // because otherwise it won't be possible to write the code of applier methods that make sense to the
-         // aggregate.
+         // because otherwise it won't be possible to write the code of applier methods that make
+        // sense to the aggregate.
         final S newState = (S) getBuilder().build();
```

Comment:
Please reformat comments. Set the second line under the first. 
It looks better as for me:
 `// because otherwise it won't be possible to write the code `
` // of applier methods that make sense to the aggregate.`